### PR TITLE
Documentation for PinnedPublicKey #575 

### PR DESCRIPTION
**Pull request by alysondeives**
Documentation for PR #575

### CODE REVIEW

File: advanced-usage.md [hunk 1]
```diff
@@ -614,14 +614,57 @@ If you wish to connect to a server that uses a self-signed certificate, you can
 * `VerifyPeer`: the authenticity of the peer's certificate.
 * `VerifyStatus`: the status of the server cert using the "Certificate Status Request" TLS extension (aka. OCSP stapling). (libcurl 7.41.0)
 
+### Pinned Public Key
+
+When negotiating a TLS or SSL connection, the server sends a certificate indicating its identity.
+A public key is extracted from this certificate and if it does not exactly match the public key provided to this option,
+`libcurl` will abort the connection before sending or receiving any data.
```
Comment (COM8) [inline advanced-usage.md:621]:
```suggestion
`libcurl` (and therefore `CPR`) will abort the connection before sending or receiving any data.
```

File: advanced-usage.md [hunk 2]
```diff
@@ -614,14 +614,57 @@ If you wish to connect to a server that uses a self-signed certificate, you can
...
+cpr::SslOptions sslOpts = cpr::Ssl(ssl::PinnedPublicKey{"pubkey.pem"});
+/* OR
+cpr::SslOptions sslOpts = cpr::Ssl(ssl::PinnedPublicKey{"sha256//J0dKy1gw45muM4o/vm/tskFQ2BWudtp9XLxaW7OtowQ="});
+*/
+cpr::Response r = cpr::Get(cpr::Url{"https://www.httpbin.org/get"}, sslOpts);
+```
+
+If you do not have the server's public key file you can extract it from the server's certificate:
+
+``` bash
```
Comment (COM8) [inline advanced-usage.md:637]: Nice addition 👍.

**Review by COM8 — CHANGES_REQUESTED** [review]
Nice documentation.

**Comment by alysondeives** [comment]
> Nice documentation.

Thanks. I extracted it from curl doc

https://curl.se/libcurl/c/CURLOPT_PINNEDPUBLICKEY.html

**Review by COM8 — APPROVED** [review]

**Pull request merged**